packages/react-components/src/components/time-sync/viewportAdapter.tsx

```diff
@@ -147,3 +147,31 @@ export const getViewportStartOnBackwardRelative = (value: DateRangePickerProps.R
   }
   return newStart;
 };
+
+export const getViewportStartOnFowardRelative = (value: DateRangePickerProps.RelativeValue): Date => {
```

Comment:
this and the above getViewportStartOnBackwardRelative loooks the same ?

Reply:
probably worth just having the something like , `value.amount  * forward ? 1 : -1 `

Reply:
got it